client/a8c-for-agencies/sections/overview/sidebar/contact-support/style.scss

```diff
@@ -0,0 +1,30 @@
+.overview__contact-support {
+	padding: 8px;
+	border-radius: 4px;
+
+	.foldable-card__header {
+		padding-left: 12px;
+		padding-right: 12px;
+	}
+
+	#a4a-contact-support-jetpack-icon svg {
+		color: var(--color-jetpack);
+	}
+
+	#a4a-contact-support-woocommerce-icon svg {
+		color: var(--color-woocommerce);
+	}
+
+	#a4a-contact-support-pressable-icon svg {
+		margin: 3px;
+	}
+
+	#a4a-contact-support-wordpress-icon svg {
+		color: var(--color-wordpress-com);
+		margin: 3px;
+	}
```

Comment:
Using IDs here to apply custom colors in the `SidebarNavigatorMenuItem` icons, as there is no support for a custom `className` prop, and any `className` on the original icon component is not used in the actual rendering of the icon.